test/rokul-runnings-unit-tests.ts

```diff
@@ -123,6 +133,22 @@ describe('Rokul Runnings Unit tests', function() {
     expect(await rr.pressBtn({ keyPress: Buttons.up, delayInMillis: 100 })).to.eql(200);
   });
 
+  it('Should Verify Button is Pressed Down', async function() {
+    nock(baseURL)
+      .post(`/keydown/up`)
```

Comment:
should this be a regex?

Reply:
These don't need to be regex, but for purposes of similarity with the above `input` calls, I'll change them to regex.

(In these specific cases, since I'm only calling a singular button, and I'm mocking that specific endpoint for that button, I'm good.)

Reply:
Addressed in ae1d365